navigation/ca_move_base/launch/move_base_flex.launch

```diff
@@ -0,0 +1,56 @@
+<launch>
+  <arg name="robot_id"  default="$(optenv ID 1)"      doc="Unique identifier of the robot [1-Inf.)"/>
+  <arg name="ns"        value="create$(arg robot_id)" doc="Namespace of the robot. By default: create1"/>
+  <arg name="tf_prefix" value="$(arg ns)_tf"          doc="Tf prefix"/>
+
+  <arg name="laser" default="$(optenv LASER rplidar)" doc="Laser for mapping the environment"/>
+
+  <arg name="global_planner" value="$(optenv GLOBAL_PLANNER gbp)"/>
```

Comment:
Have you tried mbf with other planners?

Reply:
Yes, when I did this last update to get planners from env I tested with all the local and global planners in the package to make sure none had issues.

Reply:
I tested them and are working well!